### DISCUSSION TITLE
Black-box attack SPSA

### DISCUSSION THREAD

**Discussion by minkyu-choi04** [Trusted-AI/adversarial-robustness-toolbox#1230] [Ideas]
I believe SPSA (https://arxiv.org/pdf/1802.05666.pdf) is not yet in the toolbox. SPSA is widely used black box attack and it would be great to include it in ART. I wonder if ART has a plan for it.

**Comment by beat-buesser** [comment]
Hi @minkyu-choi04 Thank you very much for your interest in ART and proposing new tools! I think you are right, SPSA is not yet available in ART as black-box evasion tool and that it would be a great addition! Would you be interested to implement and contribute it to ART? We would be very happy to provide help and support!
